react/components/SideBar.tsx

```diff
@@ -19,9 +18,10 @@ interface Chapter {
 }
 
 const SideBar: FunctionComponent = () => {
-  const { appName } = useAppNameAndFile()
+  const appName =
+    useAppNameAndFile().appName || 'vtex.io-documentation@0.0.2-beta'
```

Comment:
is this right?

Reply:
Yep hahaha that `useAppNameAndFile().appName` could be `undefined`, and then the `appName` variable would be assigned `'vtex.io-documentation@0.0.2-beta'` as it's value.
This `beta` part is strange, I gotta admit hahaha but I'm going to release a stable version on that `vtex.io-documentation` app so that I can just write

```
const appName =
    useAppNameAndFile().appName || 'vtex.io-documentation@0.x'
```